src/rtext.c

```diff
@@ -1069,13 +1069,14 @@ bool ExportFontAsCode(Font font, const char *fileName)
 // NOTE: Uses default font
 void DrawFPS(int posX, int posY)
 {
-    Color color = LIME;                         // Good FPS
```

Comment:
I prefer the variable initialized at init and only updated when required.

Reply:
In that case, then the code will assign the value twice wasting cycles.

Reply:
I think it's better for readability, I use that approach in most functions. I prefer to keep the consistency between functions despite the additional cost.

Reply:
Alright, next iter.